wherobots/db/cursor.py

```diff
@@ -43,8 +43,32 @@ def __get_results(self) -> Optional[list[Any]]:
         result = self.__queue.get()
         if isinstance(result, DatabaseError):
             raise result
+        type_map = {
+            'object': 'STRING',
+            'int64': 'NUMBER',
+            'float64': 'NUMBER',
+            'datetime64[ns]': 'DATETIME',
+            'timedelta[ns]': 'DATETIME',
+            'bool': 'NUMBER',  # Assuming boolean is stored as number
+            'bytes': 'BINARY',
+        }
```

Comment:
If this is a constant, let's extract it out of this class into a module-level `_TYPE_MAP` constant.